README.md

```diff
@@ -1,2 +1,48 @@
-# apisix-seed
-Do service discovery on the CP side
+# APISIX-Seed for Apache APISIX
+[![Go Report Card](https://goreportcard.com/badge/github.com/api7/apisix-seed)](https://goreportcard.com/report/github.com/api7/apisix-seed)
+[![Build Status](https://github.com/api7/apisix-seed/workflows/unit-test-ci/badge.svg?branch=main)](https://github.com/api7/apisix-seed/actions)
+[![Codecov](https://codecov.io/gh/api7/apisix-seed/branch/main/graph/badge.svg)](https://codecov.io/gh/api7/apisix-seed)
+
+Do service discovery for Apache APISIX on the Control Plane
+
+# What's APISIX-Seed
+[Apache APISIX](https://github.com/apache/apisix) is a dynamic, real-time, high-performance API gateway.
+
+In terms of architecture design, Apache APISIX is divided into two parts: data plane and control plane. The data plane is Apache APISIX itself, which is the component of the traffic proxy and offers many full-featured plugins covering areas such as authentication, security, traffic control, serverless, analytics & monitoring, transformations and logging.
+The control plane is mainly used to manage routing, and implement the configuration center through etcd.
+
+For cloud-native gateways, it is necessary to dynamically obtain the latest service instance information (service discovery) through the service registry. Currently, Apache APISIX already supports [service discovery](https://github.com/apache/apisix/blob/master/docs/en/latest/discovery.md) in the data plane.
+
+This project is a component of Apache APISIX to implement service discovery in the control plane. It supports cluster deployment. At present, we have supported zookeeper and nacos. We will also support more service registries.
+
+The following figure is the topology diagram of APISIX-Seed deployment.
+
+![apisix-seed overview](docs/assets/images/apisix-seed%20overview.png)
+
+# Why APISIX-Seed
+- Network topology becomes simpler
+
+> Apache APISIX does not need to maintain a network connection with each registry, and only needs to pay attention to the configuration information in Etcd. This will greatly simplify the network topology.
```

Comment:
```suggestion
> Apache APISIX does not need to maintain a network connection with each registry, and only needs to pay attention to the configuration information in etcd. This will greatly simplify the network topology.
```